app/Http/Controllers/Api/ApiController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+
+class ApiController extends Controller
+{
+    /**
+     * Return an api response
+     * This method is used when an object ir returned
+     * @param Array $data
+     *
+     * @return \Illuminate\Http\JsonResponse
+     */
+    protected function objectResponse($data)
+    {
+        return response()->json($data);
+    }
+
+    /**
+     * Return an api response with pagination
+     *
+     * @param \Illuminate\Database\Eloquent\Collection $data
+     *
+     * @return \Illuminate\Http\JsonResponse
+     */
+    protected function paginateResponse($data)
+    {
+        $per_page = (request()->per_page ? request()->per_page : 15);
```

Comment:
Definitely think 50 should be the default maybe?

Reply:
Sounds good, i'll update accordingly